app/services/iiif_print/derivative_rodeo_service.rb

```diff
@@ -97,11 +99,13 @@ def self.derivative_rodeo_uri(file_set:, filename: nil, extension: nil)
       # original.
       extension ||= File.extname(filename)
       extension = ".#{extension}" unless extension.start_with?(".")
-      basename = File.basename(filename, extension)
+
+      # We want to strip off the extension of the given filename.
+      basename = File.basename(filename, File.extname(filename))
```

Comment:
I dont understand how this change commplishes the described behavior or if the `extension` variable is still used anywhere. Previously `extension` was set to `File.extname(filename)` and then it gets modified by adding in the `.`. Now we just use `File.extname(filename)` directly with out th `.` work. Is `extension` still needed in this case? could original line 99 just be removed and accomplish the same thing?